DC-322-L-1 Web Engineering Lab/main.py

```diff
@@ -0,0 +1,174 @@
+        It checks if the entered account number already exists. If it does, a message is displayed.
+        If the account number is unique, the function creates a new account with an initial balance of 100
+        and stores the account details in a data file.
+
+        Returns:
+        None
+        """
+
+    accountDetails = []
+    utils.registerHeader()
+
+    while True:
+        accountNumber = utils.validateUserInput(
+            0, 999, "Enter 3 digit Account Number")
+        accountPassword = utils.validateUserInput(
+            0, 9999, "Enter 4 digit password")
+
+        accountNumber = "ATM" + str(accountNumber).zfill(3)
+        if utils.isAccountExist(accountNumber):
+            utils.accountExistMsg()
+        else:
+            accountDetails = [accountNumber, accountPassword, 100]
+            utils.writeDataToFile(accountDetails)
+            break
+
+
+def checkBalance(accountNumber):
+    """
+        Allows a user to check his/her balance.
+
+        This function will take account number as an argument
+        read data from file and on through iterating that data
+        check the balance of the user
+
+        Returns:
+        account balance
+    """
+    atmData = utils.readDataFromFile()
+
+    for account in atmData:
+        if accountNumber == account["account_number"]:
+            return account["account_balance"]
+
+
+def withdrawAmount(accountNumber):
+    """
+        Allows a user to withdraw his/her funds.
+
+        This function will take account number as an argument
+        read data from file and on through iterating that data
+        and check if the withdraw ammount is equal or greater than
+        the balance then upate the object.
+
+        Returns:
+        updated atmData
+    """
+
+    atmData = utils.readDataFromFile()
+
+    withdrawAmount = utils.validateUserInput(
+        0, 99999999, "Enter Amount to withdraw: ")
+
+    for account in atmData:
+        if accountNumber == account["account_number"]:
+            account_balance = int(account["account_balance"])
+            if withdrawAmount <= account_balance:
+                account["account_balance"] = str(
+                    account_balance - withdrawAmount)
+                return atmData
+
+
+def depositAmmount(accountNumber):
+    """
+        Allows a user to Deposit his/her funds.
+
+        This function will take account number as an argument
+        read data from file and on through iterating that data
+        and check if the withdraw ammount is equal or greater than
+        the balance then upate the object.
+
+        Returns:
+        updated atmData
+    """
+
+    atmData = utils.readDataFromFile()
+
+    depositAmmount = utils.validateUserInput(
+        0, 99999999, "Enter Amount to withdraw: ")
+
+    for account in atmData:
+        if accountNumber == account["account_number"]:
+            account_balance = int(account["account_balance"])
+            account["account_balance"] = str(
```

Comment:
avoid such indentations, which create confusion, keep it at the same line. CHANGE REST OF SUCH INCONSISTENCIES TOO.